Makefile

```diff
@@ -142,6 +142,11 @@ test-docker-py: build ## run the docker-py tests
 test-integration-cli: build ## run the integration tests
 	$(DOCKER_RUN_DOCKER) hack/make.sh build-integration-test-binary dynbinary test-integration-cli
 
+test-integration-cli-on-swarm: ## run the integration tests in parall across a Swarm cluster
+# For ease of passing arguments (e.g. --replicas 3), the script is currently not started via Makefile
+	echo "Please use contrib/integration-cli-on-swarm/integration-cli-on-swarm.sh"
+	contrib/integration-cli-on-swarm/integration-cli-on-swarm.sh --help
```

Comment:
Thank you for the feedback.
I'll try to provide functional `make` target (or just remove at the moment and revive in a future PR)